UPGRADING.md

```diff
@@ -1,3 +1,276 @@
+
+For more details on the release, please see the [release notes][v2].
+
+## Release Binary
+
+Please use the correct release binary: `v2.0.0`.
+
+## Instructions
+
+- [Upgrading AtomOne](#upgrading-atomeone)
+  - [Release Binary](#release-binary)
+  - [Instructions](#instructions)
+  - [On-chain governance proposal attains consensus](#on-chain-governance-proposal-attains-consensus)
+  - [Upgrade date](#upgrade-date)
+  - [Preparing for the upgrade](#preparing-for-the-upgrade)
+    - [System requirements](#system-requirements)
+    - [Backups](#backups)
+    - [Testing](#testing)
+    - [Current runtime](#current-runtime)
+    - [Target runtime](#target-runtime)
+  - [Upgrade steps](#upgrade-steps)
+    - [`minimum-gas-prices` change](#minimum-gas-prices-change)
+    - [Method I: Manual Upgrade](#method-i-manual-upgrade)
+    - [Method II: Upgrade using Cosmovisor](#method-ii-upgrade-using-cosmovisor)
+      - [Manually preparing the binary](#manually-preparing-the-binary)
+        - [Preparation](#preparation)
+      - [Auto-Downloading the AtomOne binary](#auto-downloading-the-gaia-binary)
+  - [Expected upgrade result](#expected-upgrade-result)
+  - [Upgrade duration](#upgrade-duration)
+  - [Rollback plan](#rollback-plan)
+  - [Communications](#communications)
+  - [Risks](#risks)
+
+## On-chain governance proposal attains consensus
+
+Once a software upgrade governance proposal is submitted to the Cosmos Hub,
+both a reference to this proposal and an `UPGRADE_HEIGHT` are added to the
+[release notes][v2].
+If and when this proposal reaches consensus, the upgrade height will be used to
+halt the "old" chain binaries. You can check the proposal on one of the block
+explorers or using the `atomoned` CLI tool.
+Neither core developers nor core funding entities control the governance.
+
+## Upgrade date
+
+The date/time of the upgrade is subject to change as blocks are not generated
+at a constant interval. You can stay up-to-date by checking the estimated
+estimated time until the block is produced one of the block explorers (e.g.
+https://www.mintscan.io/atomone/blocks/`UPGRADE_HEIGHT`).
+
+## Preparing for the upgrade
+
+### System requirements
+
+### Backups
+
+Prior to the upgrade, validators are encouraged to take a full data snapshot.
+Snapshotting depends heavily on infrastructure, but generally this can be done
+by backing up the `.atomone` directory.
+If you use Cosmovisor to upgrade, by default, Cosmovisor will backup your data
+upon upgrade. See below [upgrade using cosmovisor](#method-ii-upgrade-using-cosmovisor)
+section.
+
+It is critically important for validator operators to back-up the
+`.atomone/data/priv_validator_state.json` file after stopping the atomoned
+process. This file is updated every block as your validator participates in
+consensus rounds. It is a critical file needed to prevent double-signing, in
+case the upgrade fails and the previous chain needs to be restarted.
+
+### Testing
+
+XXX UPDATE?
```

Comment:
Let's revisit as we make plans to run a testnet. May be worth removing the text from this section so we can unblock this PR (and update later) or wait until we have a clearer picture

Reply:
Ok for addressing that later and not block this PR, I removed that section [4f21380](https://github.com/atomone-hub/atomone/pull/51/commits/4f213805e5e8f5d8300e8733b2a288450083b7f5)